lib/install/management/create_test.go

```diff
@@ -83,7 +83,18 @@ func TestMain(t *testing.T) {
 		errConf.VolumeLocations = make(map[string]*url.URL)
 		errConf.VolumeLocations["volume-store"], _ = url.Parse("ds://store_not_exist/volumes/test")
 		testCreateVolumeStores(op, validator.Session, errConf, true, t)
+
+		// FIXME: (pull vic/7088) have to make another VCH config from validator for negative test case and cleanup test
+		// If we re-use the previous validator like we did in the earlier test (*errConf = *conf), it's not a deep copy of conf
+		// This conf will get modified by appliance creation and cleanup test, and we can't test create appliance in positive case
+		// The other way around, if we test positive case first, the VCH data and session data are modified, so we are not able to test the negative case
+		conf2, err := validator.Validate(op, input)
+		conf2.ImageStores[0].Host = "http://non-exist"
```

Comment:
I think this will cause a validator failure meaning we'll never try creating the resource pool or appliance.
Please confirm that you see the expected output when running the test.

Reply:
This `conf2` produced from validation is identical to the `conf` used in existing tests. For some reason, in producing `conf` in existing tests, the validation failed from the output from running the unit test, but a config is successfully created, and I do see the expected log messages from the unit test output when running it. Also checked with manual testing.